lightning/src/onion_message/messenger.rs

```diff
@@ -432,7 +433,7 @@ pub trait MessageRouter {
 	fn create_blinded_paths<
 		T: secp256k1::Signing + secp256k1::Verification
 	>(
-		&self, recipient: PublicKey, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
+		&self, recipient: PublicKey, recipient_tlvs: Option<RecipientData>, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
```

Comment:
Here and elsewhere: s/recipient_tlvs/recipient_data

Reply:
Done! [8e4e1aa](https://github.com/lightningdevkit/rust-lightning/commit/8e4e1aabadb5fda9f4d0ec2c845263c94c273643)
Thanks!